src/main/java/hudson/scm/CredentialsSVNAuthenticationProviderImpl.java

```diff
@@ -270,11 +292,20 @@ public void reset() {
 
         private final Credentials defaultCredentials;
         private final Map<String, Credentials> credentialsByRealm;
+        private final TaskListener listener;
 
         public RemotableSVNAuthenticationBuilderProvider(Credentials defaultCredentials,
-                                                         Map<String, Credentials> credentialsByRealm) {
+                                                         Map<String, Credentials> credentialsByRealm,
+                                                         TaskListener listener) {
             this.defaultCredentials = defaultCredentials;
             this.credentialsByRealm = credentialsByRealm;
+            this.listener = listener == TaskListener.NULL ? null : listener;
```

Comment:
> Why would we need it?

Because `NULL` is not `Serializable` I am afraid.



> One can copy-paste his own equivalent of `TaskListener.NULL`, which will have a different behavior then

Assuming it is `Serializable`, it should have the same behavior—nothing will be printed.